mitiq/zne/inference.py

```diff
@@ -920,6 +920,101 @@ def reduce(self) -> float:
         return self._zne_limit
 
 
+class RungeFactory(BatchedFactory):
+    """Factory object implementing another version of Richardson extrapolation.
+    In this version the original set of nodes (scale_factors) are mapped to set
+    of fake nodes known as Chebyshev-Lobatto points. This method gives a better
+    interpolation when there are a lot of nodes, especially close to the
+    edges of the domain.
+
+    Reference:
+    S. De Marchi, F. Marchetti, E. Perracchione, D. Poggiali,
+    Polynomial interpolation via mapped bases without resampling link, JCAM.
```

Comment:
Did you mean to add a link here? Preferably to open-access, though I only found this with a quick search: https://www.sciencedirect.com/science/article/abs/pii/S0377042719303449

Reply:
Added